src/main/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusValidation.java

```diff
@@ -122,6 +125,13 @@ private static void validateTable(
         GTFSFeed gtfsFeed
     ) throws IOException {
         String tableId = specTable.get("id").asText();
+        Boolean tableIsDirections = tableId.equals("directions");
+
+        Map<String, Route> gtfsRoutes = new HashMap<>();
```

Comment:
The Route objects are not required. This should just be `Set<String> gtfsRoutes = null;`. Then update where this is referenced.

Reply:
Moved to a set in https://github.com/ibi-group/datatools-server/pull/559/commits/cf83716376c894677cddf3752eb439962f1f4f62